mule-gateway/modules/ROOT/pages/mule-gateway-proxy-config.adoc

```diff
@@ -0,0 +1,95 @@
+= Connect the Mule Gateway Through a Proxy Server
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can configure the Mule Gateway to route traffic through the proxy server to API Manager if your Mule Runtime engine runs inside a firewall that restricts external communication through a proxy server.
+
+Mule Gateway supports Basic, NTLM, and Kerberos v5 (via SPNEGO) authentication schemes for proxied communication.
+
+
+== Configure the Mule Gateway to Connect Through a Proxy Server
+
+To configure Mule Gateway to connect through a proxy server, you must set up your proxy server configuration in a `$MULE_HOME/conf/wrapper.conf` file:
+
+
+
+To configure the proxy server in the `$MULE_HOME/conf/wrapper.conf` file, add your proxy server information to the following properties:
+
+* `wrapper.java.additional.<n>=-Danypoint.platform.proxy_host={hostname}`
+* `wrapper.java.additional.<n>=-Danypoint.platform.proxy_port={port}`
+* `wrapper.java.additional.<n>=-Danypoint.platform.proxy_username={username}`
+* `wrapper.java.additional.<n>=-Danypoint.platform.proxy_password={password}`
+
+[IMPORTANT]
+These parameters are additional options to pass to Java during launch. The <n> element refers to the number of additional parameters in the configuration, starting from `1` and incrementing sequentially without any gaps. 
+
+With the provided credentials, Mule Gateway will prioritize Basic Authentication over NTLM. If Basic Authentication fails, the Gateway will attempt to authenticate with NTLM. +
+For Kerberos authentication, proceed to the next section.
```

Comment:
remove

Reply:
Remove the `For Kerberos authentication, proceed to the next section.`?